src/js/disability-benefits/containers/DisabilityBenefitsApp.jsx

```diff
@@ -45,7 +45,7 @@ class DisabilityBenefitsApp extends React.Component {
     const { available, synced, authorized } = this.props;
 
     return (
-      <RequiredLoginView authRequired={3} serviceRequired={"disability-benefits"}>
+      <RequiredLoginView authRequired={3} serviceRequired={"evss-claims"}>
```

Comment:
We are sending both service names
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/serializers/user_serializer.rb#L36

We also had both routes active, I have another PR for the API that will remove the route and old service name after this is merged